### PR TITLE
docs: update plugins information

### PR DESCRIPTION
**Pull request by K3vinF**
I found this section quite confusing. I have tried to make a few improvements:

- Added basic explanation of pwa plugins ;
- Better explain relation between Shopware plugin and pwa plugin;

### Checklist

- [ ] the [list of features](docs/guide/FEATURELIST.md) is updated (if it's a feature and it's relevant)
- [ ] I followed [contributing](https://github.com/DivanteLtd/shopware-pwa/blob/master/CONTRIBUTING.md) guidelines

### CODE REVIEW

File: docs/landing/concepts/plugins.md [hunk 1]
```diff
@@ -1,10 +1,14 @@
...
+(Shopware-pwa) plugins can be used to provide content to slots, or provide additional layouts or pages.
+
+Plugins can be provided from the connected Shopware instance or can be provided locally from inside the PWA.
+
+`npx @shopware-pwa/cli init` will take the resources from each **Shopware** plugin `[plugin]/src/Resources/app/pwa` directory in your connected Shopware instance and copy it to your **PWA** project. Plugin installation date determines whether it's "on top" of other plugins (newest installation date means that plugin is on top if there are more than one plugin injected to theme slot)
 
 ## Creating plugin
 
-Plugin development can be easily made locally. First, we need to choose where would we want to plug into. To see all available plugin slots we may want to select `yes` during `npx @shopware-pwa/cli init` command when it asks to allow dev mode.
+Plugin development can be easily made locally inside the PWA too. If we want to provide content to a slot, we first need to choose where would we want to plug it into. To see all available plugin slots we may want to select `yes` during `npx @shopware-pwa/cli init` command when it asks to allow dev mode.
```
Comment (patzick) [inline docs/landing/concepts/plugins.md:11]:
```suggestion
Plugin development can be easily made locally inside the PWA too. If we want to provide content to a slot, we first need to choose where would we want to plug it into. To see all available plugin slots we can look into Vue devtools. The default theme provides some slots out of the box, but you can also use custom themes or add slots on your own.
```

**Review by patzick — COMMENTED** [review]
Thank you for that!
One suggestion to see plugin slots - option to see them is currently turned off in CLI, as it was confusing question for the new people and if you have nested slots then they wouldn't be visible - Vue devtools are better tool for that :)

**Review by mkucmus — APPROVED** [review]
Another improvement to make it clear 💙

**Review by patzick — APPROVED** [review]
👌  thank you!

**Pull request merged**